### PR TITLE
Release 3.0.2

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

**Review by RonnyPfannschmidt — APPROVED** [review]
:boom:

**Comment by nicoddemus** [comment]
Oh oh.

It published `pytest_xdist-0.0.0-py3-none-any.whl`... setuptools-scm did not kick-in?

**Comment by nicoddemus** [comment]
Weird, locally it generated correctly the package as 3.0.0... any ideas @RonnyPfannschmidt ?

**Comment by nicoddemus** [comment]
Hmm seems like checkout@v2 does not check all tags:

https://github.com/pytest-dev/pytest-xdist/actions/runs/3320401672/jobs/5486795138

```
/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +192193197bee27228e1e4b44855b3777397e31e5:refs/tags/v3.0.0
```

(See `--no-tags` above)

Will change the action to checkout all tags.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
             python: "3.8"
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```
Comment (RonnyPfannschmidt) [inline .github/workflows/main.yml:43]: lets set the depth here as well

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus https://github.com/actions/checkout/issues/249 has been open sine a while now, they just keep avoiding adding a sensible way

File: .github/workflows/main.yml (hunk 1 above)
Comment (nicoddemus) [reply .github/workflows/main.yml:43]: Done

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus yikes, new issue it seems, we may need the output with SETUPTOOLS_SCM_DEBUG=1 of the wheel build

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus this is a completely new one for me as well - no idea yet whats wrong

**Comment by mdmintz** [comment]
Might be unrelated, but ``pytest`` just removed ``py`` as a dependency, yet ``pytest-xdist`` has ``pytest-forked`` as a dependency, which has ``py`` as a dependency.

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus i'll have a bugfix in about 10

**Comment by nicoddemus** [comment]
Yay finally, thanks @RonnyPfannschmidt for the quick fix. 👍

**Pull request merged**